test/integration/command_test.exs

```diff
@@ -13,6 +13,11 @@ defmodule Integration.CommandTest do
     assert response["data"]["response"] == "req-opt response"
   end
 
+  test "running a command with a bad template", %{user: user} do
+    response = send_message(user, "@bot: operable:bad-template")
+    assert response["data"]["response"] == "@vanstee Whoops! An error occurred. There was an error rendering the template 'badtemplate' for the adapter 'test': %Protocol.UndefinedError{description: nil, protocol: String.Chars, value: %{\"bad\" => %{\"foo\" => \"bar\"}}}"
```

Comment:
Any way we can produce a more informative error message than a `Protocol.UndefinedError`?